hedera-node/hedera-app/src/main/java/com/hedera/node/app/roster/RosterServiceImpl.java

```diff
@@ -29,13 +30,11 @@
  * Not exposed outside `hedera-app`.
  */
 public class RosterServiceImpl implements Service {
-    /** The name of this service */
-    public static final String NAME = "RosterService";
 
     @NonNull
     @Override
     public String getServiceName() {
-        return NAME;
+        return RosterStateId.NAME;
```

Comment:
There's a need to access the RosterService's name from both the app and platform modules.  The easiest way to do that without duplicating the service name is to define it in some common module. Other services duplicate the same name in places the service definition cannot be accessed. I prefer to have it somewhere it can be referenced instead but if you prefer we duplicate the service names, I'm ok to do it.